discordkit/utils/ClientGen.ts

```
@@ -0,0 +1,62 @@
+            if (error instanceof Error) {
+              console.error("Error stack:", error.stack);
+            }
+            return new Error(
+              `Failed to build client: ${error instanceof Error ? error.message : String(error)}`,
+            );
+          },
+        });
+      },
+    };
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)